include/lemlib/timer.hpp

```diff
@@ -10,25 +11,25 @@ class Timer {
          *
          * @param time how long to wait, in milliseconds
```

Comment:
```suggestion
         * @param time how long to wait
```
get rid of unit in description